about.html

```diff
@@ -54,33 +54,41 @@ <h2 id="logo-title">Creative Commons Global Summit 2015 Logo</h2>
                                             process, the logo from Naresh Agrawal won the prize.
                                             After consultation with Mr. Agrawal we optimized the prize-winning-logo
                                             for this year´s summit as following:</p>
-        <div id="logo-about"><a href="index_home-html" alt=""><img src="images/logo-summit.jpg" alt=""></a></div>
+        <div id="logo-about"><a href="index.html" alt=""><img src="images/logo-summit.jpg" alt=""></a></div>
```

Comment:
- [OPTIONAL] I noticed a bit of horizontal scroll on the mobile version. It can be fixed by adding `width: 100%` to the `img` for mobile version. Please consider fixing this ✨🛠